src/util/PeraWalletConnectError.ts

```diff
@@ -1,17 +1,26 @@
 interface PeraWalletConnectErrorData {
   type:
-    | "SIGN_TRANSACTIONS"
+    | "MESSAGE_NOT_RECEIVED"
+    | "OPERATION_CANCELLED"
+
+    // Connect
+    | "CONNECT_MODAL_CLOSED"
+    | "CONNECT_CANCELLED"
+    | "CONNECT_NETWORK_MISMATCH"
+
+    // Reconnect
     | "SESSION_DISCONNECTED"
     | "SESSION_UPDATE"
     | "SESSION_CONNECT"
     | "SESSION_RECONNECT"
-    | "CONNECT_MODAL_CLOSED"
-    | "CONNECT_CANCELLED"
+
+    // Sign
+    | "SIGN_TRANSACTIONS"
     | "SIGN_TXN_CANCELLED"
-    | "CONNECT_NETWORK_MISMATCH"
     | "SIGN_TXN_NETWORK_MISMATCH"
-    | "MESSAGE_NOT_RECEIVED"
-    | "OPERATION_CANCELLED";
+    | "SIGN_DATA_CANCELLED"
+    | "SIGN_DATA_NETWORK_MISMATCH"
```

Comment:
we might not need this, can we use SIGN_DATA_NETWORK_MISMATCH?
error message looks generic